modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,439 @@
+
+=== Schema types translation
+
+Redpanda supports direct translations of the following types to Iceberg value domains:
+
+[tabs]
+======
+Avro::
++
+--
+|===
+| Avro type | Iceberg type
+
+| boolean | boolean
+| int | int
+| long | long
+| float | float
+| double | double
+| bytes | binary
+| string | string
+| record | struct
+| array | list
+| maps | list
+| fixed | fixed
+| decimal | decimal
+| uuid | uuid
+| date | date
+| time | time
+| timestamp | timestamp
+|===
+
+* Different flavors of time (such as `time-millis`) and timestamp (such as `timestamp-millis`) types are translated to the same Iceberg `time` and `timestamp` types respectively.
+* Avro unions are flattened to Iceberg structs with optional fields:
+** For example, the union `["int", "long", "float"]` is represented as an Iceberg struct `struct<0 INT NULLABLE, 1 LONG  NULLABLE, 2 FLOAT NULLABLE>`.
+** The union `["int", null, "float"]` is represented as an Iceberg struct `struct<0 INT NULLABLE, 1 FLOAT NULLABLE>`.
+* All fields are required by default (Avro always sets a default in binary representation).
+* The Avro duration logical type is ignored.
+* The Avro null type is ignored and not represented in the Iceberg schema.
+* Recursive types are not supported.
+--
+
+Protobuf::
++
+--
+|===
+| Protobuf type | Iceberg type
+
+| bool | boolean
+| double | double
+| float | float
+| int32 | int
+| sint32 | int
+| int64 | long
+| sint64 | long
+| sfixed32 | int
+| sfixed64 | int
+| string | string
+| bytes | binary
+| map | map
+|===
+
+* Repeated values are translated into Iceberg `array` types.
+* Enums are translated into Iceberg `int` types based on the integer value of the enumerated type.
+* uint32 and fixed32 are translated into Iceberg `long` types as that is the existing semantic for unsigned 32-bit values in Iceberg.
+* uint64 and fixed64 values are translated into their Base-10 string representation.
+* The timestamp well-known type in Protobuf is translated into `timestamp` in Iceberg.
+* Messages are converted into Iceberg structs.
+* Recursive types are not supported.
+--
+======
+
+== Set up catalog integration
+
+You can configure the Iceberg integration to either store the metadata in https://iceberg.apache.org/javadoc/1.5.0/org/apache/iceberg/hadoop/HadoopCatalog.html[HadoopCatalog^] format in the same object storage bucket or container, or connect to a REST-based catalog. 
+
+Set the cluster configuration property `iceberg_catalog_type` with one of the following values:
+
+* `rest`: Connect to and update an Iceberg catalog using a REST API. See the https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml[Iceberg REST Catalog API specification].
+* `object_storage`: Write catalog files to the same object storage bucket as the data files. Use the object storage URL with an Iceberg client to access the catalog and data files for your Redpanda Iceberg tables.
+
+Switching catalog types is not supported.
```

Comment:
```suggestion
Switching catalog types is not supported.
```

switching when? mid-flight? That's what I assume but we could clarify?

Reply:
Once a single topic has iceberg enabled, then you cannot change the catalog type. I don't think we enforce this